apps/api-documenter/src/documenters/YamlDocumenter.ts

```diff
@@ -490,7 +529,8 @@ export class YamlDocumenter {
     JsonFile.validateNoUndefinedMembers(dataObject);
 
     let stringified: string = yaml.safeDump(dataObject, {
-      lineWidth: 120
+      lineWidth: 120,
+      noRefs: this._config && this._config.noDuplicateEntries ? true : false
```

Comment:
Why are you enabling on `noRefs`?  Generally we've shied away from this YAML feature because it introduces reference nodes that interfere with diffs and don't seem to serve any useful purpose. (A properly defined file format normally doesn't include a significant amount of redundant data.)

Reply:
It can be safely removed I believe as it's more of a case-specific thing I tried. In our case, we are loading 4 api.json files: OUFR, styling, utilities, and merge-styles. OUFR on its hand was also re-exporting 2 of this: styling and utilities, so I ended up having lots of duplicates in the navigation. But we will handle this differently now so no need for it now. Forgot to remove it 😄

Reply:
the reference nodes that you say about are actually introduced if this feature is not toggled on.